cmake/FindCVC4.cmake

```diff
@@ -1,4 +1,8 @@
-find_path(CVC4_INCLUDE_DIR cvc4/cvc4.h)
-find_library(CVC4_LIBRARY NAMES cvc4 )
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(CVC4 DEFAULT_MSG CVC4_LIBRARY CVC4_INCLUDE_DIR)
+if (USE_CVC4)
+    find_path(CVC4_INCLUDE_DIR cvc4/cvc4.h)
+    find_library(CVC4_LIBRARY NAMES cvc4 )
+    include(FindPackageHandleStandardArgs)
+    find_package_handle_standard_args(CVC4 DEFAULT_MSG CVC4_LIBRARY CVC4_INCLUDE_DIR)
+else()
+    set(CVC4_FOUND "false")
```

Comment:
Just for purity I'd change it to `set(CVC4_FOUND FALSE)`.

Reply:
I can change them.